src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/RelevantStatementsType.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.overseasentitiesapi.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public enum RelevantStatementsType {
+    CEASED_TO_BE_REGISTRABLE_BENEFICIAL_OWNER("ceased_to_be_registrable_beneficial_owner"),
+    NO_CEASED_TO_BE_REGISTRABLE_BENEFICIAL_OWNER("no_ceased_to_be_registrable_beneficial_owner"),
+    TRUST_INVOLVED_IN_THE_OE("trust_involved_in_the_oe"),
+    NO_TRUST_INVOLVED_IN_THE_OE("no_trust_involved_in_the_oe"),
+    BECOME_OR_CEASED_BENEFICIARY_OF_A_TRUST("become_or_ceased_beneficiary_of_a_trust"),
+    NO_BECOME_OR_CEASED_BENEFICIARY_OF_A_TRUST("not_become_or_ceased_beneficiary_of_a_trust");
```

Comment:
this says `"NOT_become_or_ceased_beneficiary_of_a_trust"` sould it be `no_become...`?

Reply:
Updated, thanks